### PR TITLE
Update LN Markets  to v1.1.5

### PR DESCRIPTION
**Pull request by kibotrel**
Patchnote: [v1.1.5](https://github.com/ln-markets/umbrel/releases/tag/v1.1.5)

### CODE REVIEW

**Comment by kibotrel** [comment]
Small issue found, put this PR as Draft to prevent any problems. We're fixing it!

File: apps/lnmarkets/docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: "3.7"
 
 services:
   lnmarkets:
-    image: ghcr.io/ln-markets/umbrel:v1.1.4@sha256:d9168a2a3e1016b7cd302cde579ae3013aac57ebeaf89c44f5dbc02f604779cc
+    image: ghcr.io/ln-markets/umbrel:v1.1.5@sha256:7ef171d3e76d99706d2077b3f27b115a1dbff150b339378692d90ebe06f7761a
```
Comment (lukechilds) [inline apps/lnmarkets/docker-compose.yml:5]: Where did you get this hash from? I'm getting a different one

```
docker pull ghcr.io/ln-markets/umbrel:v1.1.5
v1.1.5: Pulling from ln-markets/umbrel
be307f383ecc: Already exists
b0e166648ff5: Pull complete
6d3f0f01fe60: Pull complete
50a2cb941575: Pull complete
4ef76faabbc6: Pull complete
6dd3f36b171a: Pull complete
ad54858c56e9: Pull complete
2ae138322051: Pull complete
d4d64d7f81fd: Pull complete
09fd198b4769: Pull complete
5ebe9aa03862: Pull complete
3ce5dd9600c6: Pull complete
Digest: sha256:94b2259e4c2445d518cabc4bd6c0519c5fae87a3fb3c0bb564522e805d63bd89
Status: Downloaded newer image for ghcr.io/ln-markets/umbrel:v1.1.5
ghcr.io/ln-markets/umbrel:v1.1.5
```

Comment (lukechilds) [reply apps/lnmarkets/docker-compose.yml:5]:
```suggestion
    image: ghcr.io/ln-markets/umbrel:v1.1.5@sha256:94b2259e4c2445d518cabc4bd6c0519c5fae87a3fb3c0bb564522e805d63bd89
```

Comment (vafanassieff) [reply apps/lnmarkets/docker-compose.yml]: I just pushed it, sorry again !

**Pull request merged**